states/MI/governor.yaml

```diff
@@ -0,0 +1,139 @@
+        - name: "ctl00$Content$OpinionFirstName"
+          selector: "#Content_OpinionFirstName"
+          value: $NAME_FIRST
+          required: true
+        - name: "ctl00$Content$OpinionLastName"
+          selector: "#Content_OpinionLastName"
+          value: $NAME_LAST
+          required: true
+        - name: "ctl00$Content$OpinionAddress"
+          selector: "#Content_OpinionAddress"
+          value: $ADDRESS_STREET
+          required: true
+        - name: "ctl00$Content$OpinionCity"
+          selector: "#Content_OpinionCity"
+          value: $ADDRESS_CITY
+          required: true
+    - select:
+        - name: "ctl00$Content$OpinionState"
+          selector: "#Content_OpinionState"
+          value: $ADDRESS_STATE_FULL
+          required: true
+          options:
+            "AL": "Alabama"
+            "AK": "Alaska"
+            "AZ": "Arizona"
+            "AR": "Arkansas"
+            "CA": "California"
+            "CO": "Colorado"
+            "CT": "Connecticut"
+            "DE": "Delaware"
+            "DC": "District of Columbia"
+            "FL": "Florida"
+            "GA": "Georgia"
+            "HI": "Hawaii"
+            "ID": "Idaho"
+            "IL": "Illinois"
+            "IN": "Indiana"
+            "IA": "Iowa"
+            "KS": "Kansas"
+            "KY": "Kentucky"
+            "LA": "Louisiana"
+            "ME": "Maine"
+            "MD": "Maryland"
+            "MA": "Massachusetts"
+            "MI": "Michigan"
+            "MN": "Minnesota"
+            "MS": "Mississippi"
+            "MO": "Missouri"
+            "MT": "Montana"
+            "NE": "Nebraska"
+            "NV": "Nevada"
+            "NH": "New Hampshire"
+            "NJ": "New Jersey"
+            "NM": "New Mexico"
+            "NY": "New York"
+            "NC": "North Carolina"
+            "ND": "North Dakota"
+            "OH": "Ohio"
+            "OK": "Oklahoma"
+            "OR": "Oregon"
+            "PA": "Pennsylvania"
+            "RI": "Rhode Island"
+            "SC": "South Carolina"
+            "SD": "South Dakota"
+            "TN": "Tennessee"
+            "TX": "Texas"
+            "UT": "Utah"
+            "VT": "Vermont"
+            "VA": "Virginia"
+            "WA": "Washington"
+            "WV": "West Virginia"
+            "WI": "Wisconsin"
+            "WY": "Wyoming"
+    - fill_in:
+        - name: "ctl00$Content$OpinionZipCode"
+          selector: "#Content_OpinionZipCode"
+          value: $ADDRESS_ZIP5
+          required: true
+        - name: "ctl00$Content$OpinionPhone"
+          selector: "#Content_OpinionPhone"
+          value: $PHONE
+          required: true
+        - name: "ctl00$Content$OpinionEmail"
+          selector: "#Content_OpinionEmail"
+          value: $EMAIL
+          required: true
+    - select:
+        - name: "ctl00$Content$OpinionIssue"
+          selector: "#Content_OpinionIssue"
+          value: "1f2eb027-7631-e111-b4af-00219b8d8c66"
```

Comment:
I think this can be "* Other / Unknown" if you want? As mentioned above I think you can use either and my personal choice in this case would be to use the human-readable one.

Reply:
Will it accept either value or text of an option list?  I only used the weirdo value because the form has that for the value.

```<option value="1492754f-49c5-e511-99db-00219b8d8c68">* Flint Resident</option>

Reply:
I think it will try both, if the `value` isn't found among the `select` `value`s. You can give it try both ways. (Or you can just go with how you have it now, it's okay as is!)

Reply:
OK, I changed it to use the more readable option. Gave it a test and it works.  Thanks for the tip.

Reply:
Cool~